src/components/Chart/Chart.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import _ from 'lodash';
+import { ChartSquare } from './ChartSquare';
+
+export const Chart = ({ props }) => {
+  return (
+    <div className="chart">
+      {_.map(props, (data, index) => {
+        const { rating, episode, season, type = 'episodeData' } = data;
```

Comment:
we wont want this default `type = 'episodeData'` anymore

we  want the data to include it